torchbenchmark/models/opacus_cifar10/__init__.py

```diff
@@ -0,0 +1,78 @@
+import torch
+import torch.optim as optim
+import torch.nn as nn
+import torchvision.models as models
+from opacus.utils.module_modification import convert_batchnorm_modules
+from opacus import PrivacyEngine
+
+from ...util.model import BenchmarkModel
+
+
+class Model(BenchmarkModel):
+    def __init__(self, device=None, jit=False):
```

Comment:
@xuzhao9 This model is differentially private training of an image classification problem. I think I should assign this the OTHER.OTHER_TASKS task due to the category being "differential privacy", what do you think?

Reply:
Yes, please set it to `OTHER.OTHER_TASKS`. We may have a better categorization later, but setting it to OTHER is good for now.

Reply:
Resolved